builtin/commit-graph.c

```diff
@@ -176,13 +177,15 @@ static int graph_write(int argc, const char **argv)
 			N_("maximum ratio between two levels of a split commit-graph")),
 		OPT_EXPIRY_DATE(0, "expire-time", &split_opts.expire_time,
 			N_("maximum number of commits in a non-base split commit-graph")),
+		OPT_BOOL(0, "progress", &opts.progress, N_("force progress reporting")),
 		OPT_END(),
 	};
 
 	split_opts.size_multiple = 2;
 	split_opts.max_commits = 0;
 	split_opts.expire_time = 0;
 
+	opts.progress = 1;
```

Comment:
super-nit: and i think i'd put this before the first split_opts.size_multiple assignment on 184.